lib/Models/Terria.ts

```diff
@@ -1709,6 +1697,32 @@ async function interpretHash(
     });
   });
 
+  if (isDefined(hashProperties.clean)) {
+    terria.initSources.splice(0, terria.initSources.length);
+  }
```

Comment:
I think we should process the `#clean` param before processing the `initFragments` otherwise a URL like `#clean&foo-fragment` will always skip the fragment no matter the order.

Reply:
Yep good idea!

Reply:
We can now combine hash params! 

http://ci.terria.io/convert-start-data/#clean&https://gist.githubusercontent.com/nf-s/e356d1e0cb88d9d4d306f9ae756de788/raw/d6d9f9f30ea1546f706e39d1704d73d381d70f2b/ods-v3.json&start=%7B%22version%22%3A%228.0.0%22%2C%22initSources%22%3A%5B%7B%22stratum%22%3A%22user%22%2C%22models%22%3A%7B%22%2F%2FVIC%22%3A%7B%22isOpen%22%3Atrue%2C%22knownContainerUniqueIds%22%3A%5B%22%2F%22%5D%2C%22type%22%3A%22group%22%7D%2C%22%2F%2FVIC%2FSouthern+Grampians+Shire%22%3A%7B%22isOpen%22%3Atrue%2C%22knownContainerUniqueIds%22%3A%5B%22%2F%2FVIC%22%5D%2C%22type%22%3A%22opendatasoft-group%22%7D%2C%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%22%3A%7B%22isOpen%22%3Atrue%2C%22knownContainerUniqueIds%22%3A%5B%22%2F%2FVIC%2FSouthern+Grampians+Shire%22%5D%2C%22type%22%3A%22opendatasoft-group%22%7D%2C%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%2Fgeo%22%3A%7B%22isOpen%22%3Atrue%2C%22knownContainerUniqueIds%22%3A%5B%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%22%5D%2C%22type%22%3A%22opendatasoft-group%22%7D%2C%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%2Fgeo%2Fparking-sensor-status%22%3A%7B%22knownContainerUniqueIds%22%3A%5B%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%2Fgeo%22%5D%2C%22type%22%3A%22opendatasoft-item%22%7D%7D%2C%22workbench%22%3A%5B%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%2Fgeo%2Fparking-sensor-status%22%5D%2C%22timeline%22%3A%5B%22%2F%2FVIC%2FSouthern+Grampians+Shire%2Ffeatures%2Fgeo%2Fparking-sensor-status%22%5D%2C%22initialCamera%22%3A%7B%22west%22%3A142.0229144245782%2C%22south%22%3A-37.74407983592333%2C%22east%22%3A142.02530459158496%2C%22north%22%3A-37.742878377646484%2C%22position%22%3A%7B%22x%22%3A-3980803.658061851%2C%22y%22%3A3107447.99971725%2C%22z%22%3A-3883080.0542477765%7D%2C%22direction%22%3A%7B%22x%22%3A0.6231680978958201%2C%22y%22%3A-0.4864501305841301%2C%22z%22%3A0.612395127527635%7D%2C%22up%22%3A%7B%22x%22%3A-0.4827325529811273%2C%22y%22%3A0.37682499188868995%2C%22z%22%3A0.7905518375036592%7D%7D%2C%22homeCamera%22%3A%7B%22west%22%3A109%2C%22south%22%3A-45%2C%22east%22%3A158%2C%22north%22%3A-8%7D%2C%22baseMaps%22%3A%7B%22defaultBaseMapId%22%3A%22basemap-bing-aerial-with-labels%22%7D%2C%22viewerMode%22%3A%223dSmooth%22%2C%22currentTime%22%3A%7B%22dayNumber%22%3A2459442%2C%22secondsOfDay%22%3A53352.123%7D%2C%22showSplitter%22%3Afalse%2C%22splitPosition%22%3A0.5%2C%22pickedFeatures%22%3A%7B%22providerCoords%22%3A%7B%7D%2C%22pickCoords%22%3A%7B%22lat%22%3A-37.74350949456132%2C%22lng%22%3A142.02381115728255%2C%22height%22%3A-0.0010348170060480901%7D%2C%22current%22%3A%7B%22hash%22%3A19407715966%7D%2C%22entities%22%3A%5B%7B%22hash%22%3A19407715966%7D%5D%7D%2C%22stories%22%3A%5B%5D%7D%5D%7D